### DISCUSSION TITLE
Possible to have an API Key?

### DISCUSSION THREAD

**Discussion by ozziestraw** [pocketbase/pocketbase#1314] [Q&A]
I plan to build my own authentication system in PHP and would like to make requests to Pocketbase directly from the backend.

Would it be possible to have an Admin API key to query the API? I plan on making all the collections Admin only

**Comment by ganigeorgiev** [comment]
At the moment this is not possible, but it may be added in the future since it was requested several times already (_or at least permanent/long lived admin associated tokens_; you can also check https://github.com/pocketbase/pocketbase/discussions/342).

For now you can authenticate as admin with the token from `POST /api/admins/auth-with-password`.
If you haven't already, you can find the "Web APIs" documentation at https://pocketbase.io/docs/api-admins/.
